lib/src/components/NxDonutChart/types.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import * as PropTypes from 'prop-types';
+
+export interface Props {
+  id?: string | null;
+  percent: number;
+  outerRadius: number;
+  innerRadius: number;
+  fillColors: string[];
```

Comment:
I question whether we want all this stuff to be configurable.  The overall size of the donut and the colors can be done in CSS and the inner proportions of the donut could just as well be an implementation detail, I would think.

Reply:
> I question whether we want all this stuff to be configurable. The overall size of the donut and the colors can be done in CSS and the inner proportions of the donut could just as well be an implementation detail, I would think.

Moved what I can to css in https://github.com/sonatype/sonatype-react-shared-components/pull/172/commits/5a3e4fd020df42fb5582f6d3dd79160c5e40120b and https://github.com/sonatype/sonatype-react-shared-components/pull/172/commits/2afc66af8da46c8fedc9a79e40683da1fa841a9d